data/map.txt

```diff
@@ -26512,7 +26512,7 @@ planet "Brass Second"
 
 planet "Bright Echo"
 	attributes folklore saryd tourism urban
-	landscape land/dmottl1
+	landscape land/snow20
```

Comment:
Hmm, snow20 doesn't really fit here either. It's a good image so I'd keep it, but maybe find something less lifeless since this is described as an urban world that's only slightly colder than the Saryd homeworld.